linera-base/src/crypto/ed25519.rs

```diff
@@ -310,7 +310,7 @@ impl Ed25519Signature {
     fn check_internal<'de, T>(
         &self,
         value: &T,
-        author: Ed25519PublicKey,
+        author: &Ed25519PublicKey,
```

Comment:
Are those not `Copy` ?

Reply:
Hmm, they are. I'd have thought clippy will complain